dali/test/python/test_dali_tf_dataset_eager.py

```diff
@@ -29,6 +30,76 @@ def test_tf_dataset_cpu():
     run_tf_dataset_eager_mode('cpu')
 
 
+def run_tf_dataset_with_fixed_input(dev, shape, value, dtype):
+    tensor = np.full(shape, value, dtype)
+    run_tf_dataset_eager_mode(dev,
+        get_pipeline_desc=external_source_tester(shape, dtype, FixedSampleIterator(tensor)),
+        to_dataset=external_source_converter_with_fixed_value(shape, dtype, tensor))
+
+def test_tf_dataset_with_fixed_input():
+    for dev in ['cpu', 'gpu']:
+        for shape in [(7, 42), (64, 64, 3), (3, 40, 40, 4)]:
+            for dtype in [np.uint8, np.int32, np.float32]:
+                for value in [42, 255]:
```

Comment:
```suggestion
                for value in [42, 255]:
```
Perhaps you don't need the whole cross product. Consider:
```
value = random.choice([42, 255])
```

Reply:
Sure